core/iwasm/fast-jit/jit_regalloc.c

```diff
@@ -542,8 +538,7 @@ allocate_hreg(RegallocContext *rc, JitReg vreg, JitInsn *insn, int distance)
     if (hreg_num == 0)
     /* Unsupported hard register kind.  */
     {
-        /* TODO: how to set error */
-        /*jit_set_error (JIT_ERROR_UNSUPPORTED_HREG);*/
+        jit_set_last_error(rc->cc, "unsupported hard register num");
```

Comment:
it should mean there is no hardware register in this `kind`.

BTW, if no hardware register, we should spill all, right?

Reply:
Yes, we may change the error message.
If all hardware registers are used, we should spill part of them out to spill area, and allocate them for new virtual registers, and in some time, spill them in.

If no hardware registers are configured, e.g. no int64 registers in x86-32, register allocator cannot work if there are JIT IRs using i64 registers. Here is to check this situation.
And normally in this case, the codegen lower pass (lower_cg) should translate int64 related IRs into int32 related IRs, e.g. (convert one i64 IR `ADD I8, I9, I10` into two or more i32 IRs, e.g. `ADD ixx, ixx, ixx`)